sql/sqlite/SQLiteParser.g4

```diff
@@ -118,7 +118,7 @@ create_table_stmt:
 column_def: column_name type_name? column_constraint*;
 
 type_name:
-	name+ (
+	name+? (
```

Comment:
Please replace with `name*`

Reply:
That wouldn't be right.  The grammar should not match `CREATE TABLE foo (bar (1) NOT NULL)`.